hadoop-plugin/src/main/groovy/com/linkedin/gradle/azkaban/AzkabanHelper.groovy

```diff
@@ -358,17 +379,21 @@ class AzkabanHelper {
       logger.lifecycle("Azkaban URL: ${azkProject.azkabanUrl}");
```

Comment:
Can you please log Azkaban Zip Task after azkabanUrl, although it is not a part of this issue.
logger.lifecycle("Azkaban Zip Task: ${azkProject.azkabanZipTask}");